typescript/src/events/userUpdate.ts

```diff
@@ -25,14 +24,14 @@ export class UserEvent extends Event {
 	}
 
 	private async processGuild(guild: Guild, user: User, previous: string, next: string) {
-		const [enabled, language] = await guild.readSettings((settings) => [
-			settings[GuildSettings.Events.MemberUserNameUpdate],
+		const [channelId, language] = await guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [logChannelId, language] = await guild.readSettings((settings) => [
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.